Cargo.toml

```diff
@@ -41,9 +41,8 @@ path = "./wgpu-hal"
 version = "0.15"
 
 [workspace.dependencies.naga]
-git = "https://github.com/gfx-rs/naga"
-rev = "f0edae8"
-version = "0.11"
```

Comment:
When you update naga after the naga PR is merged, leave the version number in place

Reply:
We landed https://github.com/gfx-rs/wgpu/pull/3451, if you rebase you won't need to change the `toml`s.